src/NBomber/Api/Builder.fs

```diff
@@ -0,0 +1,190 @@
+        Step.createPause timeSpan |> addTo scenario
+
+    member _.Pause(scenario: Scenario, getTimeSpan : unit -> TimeSpan) =
+        Step.createPause getTimeSpan |> addTo scenario
+
+    /// set warmup duration
+    [<CustomOperation "warmUp">]
+    member _.WarmUp(scenario: Scenario, time) =
+        Scenario.withWarmUpDuration time scenario
+
+    /// run without warmup
+    [<CustomOperation "noWarmUp">]
+    member _.NoWarmUp(scenario: Scenario) =
+        Scenario.withWarmUpDuration TimeSpan.Zero scenario
+
+    /// setup load simulation
+    [<CustomOperation "load">]
+    member _.Load(scenario: Scenario, simulations) =
+        Scenario.withLoadSimulations simulations scenario
+
+    /// run an action before test
+    [<CustomOperation "testInit">]
+    member _.TestInit(scenario: Scenario, init) =
+        Scenario.withTestInit init scenario
+    member _.TestInit(scenario: Scenario, init : ScenarioContext -> Task) =
+        Scenario.withTestInit (unitTask init) scenario
+
+    /// run an ation after test
+    [<CustomOperation "testClean">]
+    member _.TestClean(scenario: Scenario, clean) =
+        Scenario.withTestClean clean scenario
+    member _.TestClean(scenario: Scenario, clean : ScenarioContext -> Task) =
+        Scenario.withTestClean (unitTask clean) scenario
+
+let scenario name = ScenarioBuilder name
+
+/// performance test builder
+type RunnerBuilder() =
+    let empty = NBomberRunner.registerScenarios []
+
+    member _.Zero() = empty
+
+    member _.Yield _ = empty
+
+    member _.Run f = f
+
+    [<CustomOperation "scenarios">]
+    member _.Scenarios(ctx, scenarios) =
+        { ctx with RegisteredScenarios = scenarios }
+
+    [<CustomOperation "reportFileName">]
+    member _.ReportFileName(ctx, fileName) =
+        { ctx with ReportFileName = Some fileName }
+
+    [<CustomOperation "reports">]
+    member _.ReportFormats(ctx, reports) =
+        { ctx with ReportFormats = reports }
+
+    [<CustomOperation "noReports">]
+    member _.NoReports(ctx) =
+        { ctx with ReportFormats = [] }
+
+    [<CustomOperation "testSuite">]
+    member _.TestSuite(ctx : NBomberContext, name) =
+        { ctx with TestSuite = name }
+
+    [<CustomOperation "name">]
+    member _.Name(ctx : NBomberContext, name) =
+        printfn "runner name %s" name
+        { ctx with TestName = name }
+
+    [<CustomOperation "config">]
+    member _.Config(ctx, path) =
+        NBomberRunner.loadConfig path ctx
+
+    [<CustomOperation "configInfrastructure">]
+    member _.ConfigInfrastructure(ctx, path) =
+        NBomberRunner.loadInfraConfig path ctx
+
+    [<CustomOperation "reporter">]
+    member _.Report(ctx, reporter, interval) =
+        { ctx with ReportingSinks = reporter::ctx.ReportingSinks
+                   SendStatsInterval = interval }
+
+    [<CustomOperation "plugins">]
+    member _.Reports(ctx, plugins) =
+        { ctx with Plugins = plugins }
+
+    [<CustomOperation "application">]
```

Comment:
applicationType

Reply:
I think it could be shorter to have just `process` and `console` instea of `applicationType ApplicationType.Console`. But `process` is reserved keyword